__unported__/l10n_it_DDT_webkit/__openerp__.py

```diff
@@ -31,6 +31,7 @@
     """,
     'author': 'Agile Business Group',
     'website': 'http://www.agilebg.com',
+    'license': 'GPL-3 or any later version',
```

Comment:
@yvaucher it sounds strange this module has a license different from AGPL-3. Maybe it was only a oversight, I'll wait for @eLBati opinion considering he is author.